### PR TITLE
25_when: Ansible 2.5.x

### PR DESCRIPTION
**Pull request by jvonau**
testing

### CODE REVIEW

File: vars/default_vars.yml [hunk 1]
```diff
@@ -391,3 +391,5 @@ is_rpi: False
 is_redhat: False
 is_fedora: False
 is_centos: False
+
+#rtc_id: ds3231
```
Comment (jvonau) [inline vars/default_vars.yml:404]: Has the desired effect as I don't have a rtc attached
> TASK [1-prep : include_tasks] **************************************************
included: /opt/iiab/iiab/roles/1-prep/tasks/raspberry_pi.yml for 127.0.0.1
TASK [1-prep : Add a udev rule to transfer hwclock to system clock at dev creation] ***
skipping: [127.0.0.1]
TASK [1-prep : Check for needing to enable i2c rtc device in config.txt] *******
skipping: [127.0.0.1]
TASK [1-prep : Add a udev rule to transfer hwclock to system clock at dev creation] ***
skipping: [127.0.0.1]

**Comment by holta** [comment]
@jvonau is this this safe/wanted for IIAB 6.5?

Or better to wait for IIAB 6.6?

**Comment by holta** [comment]
Should this also be merged, building on #742?

(Or better to wait for IIAB 6.6?)

**Comment by holta** [comment]
Please use Ansible 2.5.1 to test IIAB 6.6 in coming weeks. If a rapid install of 2.5.1 is needed use:
```
pip install ansible==2.5.1
```
Changelog:
https://github.com/ansible/ansible/blob/stable-2.5/changelogs/CHANGELOG-v2.5.rst

"Future 2.5 maintenance releases will occur approximately every three weeks."

2.4 to 2.5 porting guide: https://docs.ansible.com/ansible/2.5/porting_guides/porting_guide_2.5.html

**Comment by jvonau** [comment]
no need to use pip ansible 2.5.1 debs are [available](http://ppa.launchpad.net/ansible/ansible-2.5/ubuntu/pool/main/a/ansible/)

**Comment by holta** [comment]
@jvonau suggests @tim-moody should review this PR before we move forward.

**Comment by tim-moody** [comment]
adm cons does not currently use runtags

**Comment by tim-moody** [comment]
does this force ssl for adm cons?

**Comment by jvonau** [comment]
no gui_port remains 80 until adm_cons_force_ssl is toggled

**Comment by holta** [comment]
@jvonau I need to change `iiab_base_ver: 6.5` to 6.6 — is roles/0-init/defaults/main.yml really the best place to do this?

Or should these (with `iiab_revision: 0` and/or others) be centralized somewhere?

**Comment by jvonau** [comment]
That would be the plan for now, point releases for bug fixes should
increment iiab_revision at some point

**Pull request closed without merging**